prisma/schema.sql

```diff
@@ -35,265 +48,371 @@ CREATE TYPE PhysicianStatus as ENUM('ACTIVE', 'INACTIVE');
+  disability VARCHAR(255) NOT NULL,
+  disability_certification_date DATE NOT NULL,
+  patient_condition PatientCondition NOT NULL,
+  mobility_aids MobilityAid ARRAY,
+  other_patient_condition VARCHAR(255),
+  notes TEXT,
+  physician_id INTEGER NOT NULL,
+  created_at TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  updated_at TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
 
+  PRIMARY KEY(id),
   FOREIGN KEY(physician_id) REFERENCES physicians(id)
 );
 
 -- Create applicants table
 CREATE TABLE applicants (
-  id                        SERIAL PRIMARY KEY NOT NULL,
-  first_name                VARCHAR(255) NOT NULL,
-  middle_name               VARCHAR(255),
-  last_name                 VARCHAR(255) NOT NULL,
-  date_of_birth             DATE NOT NULL,
-  gender                    Gender NOT NULL,
-  custom_gender             VARCHAR(255),
-  email                     VARCHAR(255) UNIQUE,
-  phone                     VARCHAR(50) NOT NULL,
-  province                  Province NOT NULL,
-  city                      VARCHAR(255) NOT NULL,
-  address_line_1            VARCHAR(255) NOT NULL,
-  address_line_2            VARCHAR(255),
-  postal_code               CHAR(6) NOT NULL,
-  rcd_user_id               INTEGER UNIQUE,
-  status                    ApplicantStatus,
-  inactive_reason           VARCHAR(255),
-  accepted_tos              TIMESTAMPTZ,
-  guardian_id               INTEGER UNIQUE,
-  medical_information_id    INTEGER UNIQUE NOT NULL,
-  created_at                TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
-  updated_at                TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  id SERIAL PRIMARY KEY,
+  first_name VARCHAR(255) NOT NULL,
+  middle_name VARCHAR(255),
+  last_name VARCHAR(255) NOT NULL,
+  date_of_birth DATE NOT NULL,
+  gender Gender NOT NULL,
+  other_gender VARCHAR(255),
+  phone VARCHAR(50) NOT NULL,
+  email VARCHAR(255),
+  receive_email_updates BOOLEAN NOT NULL,
+  address_line_1 VARCHAR(255) NOT NULL,
+  address_line_2 VARCHAR(255),
+  city VARCHAR(255) NOT NULL,
+  province Province NOT NULL DEFAULT 'BC',
+  country VARCHAR(255) NOT NULL DEFAULT 'Canada',
+  postal_code CHAR(6) NOT NULL,
+  rcd_user_id INTEGER UNIQUE,
+  status ApplicantStatus NOT NULL DEFAULT 'ACTIVE',
+  inactive_reason VARCHAR(255),
+  accepted_tos TIMESTAMPTZ,
+  notes TEXT,
+  guardian_id INTEGER UNIQUE,
+  medical_information_id INTEGER UNIQUE NOT NULL,
+  created_at TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  updated_at TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
 
   FOREIGN KEY(guardian_id) REFERENCES guardians(id),
   FOREIGN KEY(medical_information_id) REFERENCES medical_information(id)
 );
 
 -- Create application processing table
 CREATE TABLE application_processing (
-  id                  SERIAL PRIMARY KEY NOT NULL,
-  status              ApplicationStatus NOT NULL DEFAULT 'PENDING',
-  app_number          INTEGER UNIQUE,
-  app_holepunched     BOOLEAN NOT NULL DEFAULT false,
+  id INTEGER,
```

Comment:
Why is this an `INTEGER` instead of `SERIAL`? (same with medical information, guardian, etc.)

Reply:
Good catch. My original intention was to make the `id`s the same value as the applicant's `id`, but that would run into issues when creating an applicant. Changed relevant instances to `SERIAL PRIMARY KEY`